spec/services/notification_service_spec.rb

```diff
@@ -0,0 +1,122 @@
+describe NotificationService do
+  describe 'GET https://onesignal.com/api/v1/notifications' do
+    describe 'test all' do
+      let(:service) do
+        NotificationService.new
+      end
+
+      it 'retrieves notifications count' do
+        retrieve_notifications_mock
+        json = service.all_notifications
+        expect(JSON.parse(json.body)['total_count']).to eq '1'
+      end
+    end
+
+    describe 'test create' do
```

Comment:
The description should be `'#notify'`.Check [here](http://www.betterspecs.org/#describe)